api-reference/api-introduction/api-keys.md

```diff
@@ -1,32 +1,97 @@
 # API keys
 
-Your API requests are [authenticated](authentication.md) using the account's API key. If you do not include your key when making an API request or use an incorrect one, we return an error.
+ImageKit [authenticates](authentication.md) your API requests using your account's API keys. If you don't include your key when making an API request or use an incorrect or outdated one, ImageKit returns a `403 - Forbidden` response.
```

Comment:
I checked the code, the status code is `401` Please confirm and correct it.

Reply:
Done